From 868bb8cbc357b1b329d125ecd0cd1adf58e6c015 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Wed, 06 Nov 2019 13:44:15 +0200
Subject: [PATCH] [SW-0] hl: put assembly under proper architecture

This code is architecture specific and will fail for other ARCHs

Change-Id: I7b2b80a9647ca8079764c2f940dad173f21c40e4
---

diff --git a/common/start.S b/common/start.S
index f988923..da9ffe5 100644
--- a/common/start.S
+++ b/common/start.S
@@ -12,7 +12,10 @@
 .globl	_start
 _start:
 
+#ifdef __aarch64__
 	ldr	x0, =(_start + 0x10000)
 	bic	sp, x0, #0xf	/* 16-byte alignment for ABI compliance */
 	mov	x29, sp		/* Set FP (x29) */
+#endif
+
 	bl	preboot_main	/* Set LR (x30) */
